pyproject.toml

```diff
@@ -21,6 +21,7 @@ html2text = "*"
 psutil = "*"
 retrying = "*"
 pyaml = "^23.9.7"
+pypreprocess = "^1.1.0"
```

Comment:
Do we need to add this as a project dependency? Should be fine to keep it in the requirements.txt as you have now, since these will be installed when someone installs this llama-pack

Reply:
i think no, you've to add it at the final step, cause i thought that it's enough for requirements.txt but 2 of checks has been failed due to the auto dependency download i think relay on the poetry install only, after adding it all checks has been passed.
anyway i've kept it in requirements.txt so save the project structure as all others doing that, but to pass the checks it has to be in the project dependency.